frontend/containers/discover-map/pin-cluster/helper.ts

```diff
@@ -0,0 +1,55 @@
+import { values } from 'lodash-es';
+import Supercluster from 'supercluster';
+
+import { ProjectsMap } from 'types/project';
+
+export const PROJECT_CATEGORY_COLORS = {
```

Comment:
I don't think we should hardcode the colours here. We have the colours configured in `tailwind.config.js` and `/api/v1/enums` also returns them.

Reply:
I changed the function to use the tailwind.config.
I also changed the method to get the cluster points so I don't have to use recursion